ably/rest/auth.py

```diff
@@ -190,7 +189,7 @@ def create_token_request(self, key_id=None, key_value=None,
             token_params["nonce"] = self._random()
 
         req = {
-            "id": key_id,
+            "keyName": key_id,
```

Comment:
Yup, we will be changing the API very soon to allow both, but we will encourage use of camelCase for all params whether in the body or in the query string params.